MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1451,17 +1555,143 @@ the Select` button. When clicked it closes all the previously selected tabs -->
+         so the user can confirm if they want to actually download the file. -->
+    <string name="download_confirm_title">Would you like to download this file?</string>
+
+    <!-- This string is displayed in the "Cancel" button of the dialog displayed when a download has been requested
+         so the user can confirm if they want to actually download the file. If clicked the download is not started. -->
+    <string name="download_confirm_cancel">Cancel</string>
+
+    <!-- This string is displayed in the "Download" button of the dialog displayed when a download has been requested
+         so the user can confirm if they want to actually download the file. If clicked the download is queued. -->
+    <string name="download_confirm_download">Download</string>
+
+    <!-- This string is displayed in the confirmation dialog when a downloaded file is deleted from the downloads list. -->
+    <string name="download_delete_file_confirm_title">Delete file</string>
+
+    <!-- This string is displayed in the confirmation dialog when a downloaded file is deleted from the downloads list. -->
+    <string name="download_delete_file_confirm_body">Are you sure you want to delete this file?</string>
+
+    <!-- This string is displayed in the confirmation dialog checkbox to also delete the file from disk. -->
+    <string name="download_delete_file_confirm_checkbox">Also delete the file from disk</string>
+
+    <!-- This string is displayed in the confirmation dialog when the delete all button is clicked in the downloads list. -->
+    <string name="download_delete_all_confirm_title">Delete all downloaded files</string>
+
+    <!-- This string is displayed in the confirmation dialog when the delete all button is clicked in the downloads list. -->
+    <string name="download_delete_all_confirm_body">Are you sure you want to delete all the downloaded files?</string>
+
+    <!-- This string is displayed in the delete download confirmation dialog checkbox to also delete the file from disk. -->
+    <string name="download_delete_all_confirm_checkbox">Also delete the files from disk</string>
+
+    <!-- This string is displayed in the delete download confirmation dialog cancel button. -->
+    <string name="download_delete_confirm_cancel">Cancel</string>
+
+    <!-- This string is displayed in the confirmation dialog delete button. -->
+    <string name="download_delete_confirm_delete">Delete</string>
+
+    <!-- This string is displayed in the title of the download error dialog. -->
+    <string name="download_error_title">There was an error</string>
+
+    <!-- This string is displayed in the body of the download error dialog. -->
+    <string name="download_error_body">An error occurred while downloading %1$s. Please try again.</string>
+
+    <!-- This string is displayed in the button to dismiss the download error dialog. -->
+    <string name="download_error_ok">Ok</string>
```

Comment:
I'm almost sure that the correct spelling should be `OK`?

Reply:
Updated

Reply:
Looks like this isn't fixed yet?